blockchain/tx_pool_test.go

```diff
@@ -2277,11 +2387,12 @@ func TestTransactionsDemotionMultipleAccount(t *testing.T) {
 	// tx[6] : queue[from[2]]
 	// tx[7] : queue[from[2]]
 	// tx[7] : queue[from[2]]
+	// pool.SetBaseFee(big.NewInt(35))
 	pool.gasPrice = big.NewInt(35)
```

Comment:
here also !!! why not use `pool.SetBaseFee`? 
I think use one way to set gasPrice of the tx pool better!! (If you prefer `pool.gasPrice = `, you can change to that)

Reply:
I agree. Thansk!